src/objects.js

```diff
@@ -1,41 +1,66 @@
 const createPerson = (name, age) => {
-  // your code here
+  return {
+    name: name, 
+    age: age
+  }
 };
 
 const getName = object => {
-  // your code here
+  return object.name
 };
 
 const getProperty = (property, object) => {
-  // your code here
+  return object[property]
+  // use square bracket notation to access a property using its variable name 
 };
 
 const hasProperty = (property, object) => {
-  // your code here
+  if (object[property]){
+    return true 
+  }
+  return false
 };
 
 const isOver65 = person => {
-  // your code here
+  if (person.age > 65){ // use dot notation to retireve each key value 
+    return true
+  }
+  return false 
 };
 
 const getAges = people => {
-  // your code here
+  const ages = [];
+  for (i = 0; i < people.length; i++) { 
+    if (people[i].age) { 
+      ages.push(people[i].age);
+    }
+  }
+  return ages;
```

Comment:
Just for better readability you could also use a map function instead of a for loop :)